client/state/stats/lists/utils.js

```diff
@@ -937,66 +937,32 @@ export const normalizers = {
 	},
 
 	/**
-	 * Returns a normalized statsEmailsOpen array, ready for use in stats-module
+	 * Returns a normalized statsEmailsSummary array, ready for use in stats-module
 	 *
 	 * @param   {Object} data   Stats data
 	 * @param   {Object} query  Stats query
 	 * @param   {number} siteId  Site ID
 	 * @param   {Object} site    Site object
 	 * @returns {Array}       Normalized stats data
 	 */
-	statsEmailsOpen( data, query = {}, siteId, site ) {
-		if ( ! data || ! query.period || ! query.date ) {
+	statsEmailsSummary( data, query, siteId, site ) {
+		if ( ! data ) {
 			return [];
 		}
 
 		const emailsData = get( data, [ 'posts' ], [] );
 
-		return emailsData.map( ( { id, href, date, title, type, opens } ) => {
+		return emailsData.map( ( { id, href, date, title, type, opens, clicks } ) => {
 			const detailPage = site ? `/stats/email/opens/day/${ id }/${ site.slug }` : null;
-			return {
-				id,
-				href,
-				date,
-				label: title,
-				type,
-				value: opens || '0',
-				page: detailPage,
-				actions: [
-					{
-						type: 'link',
-						data: href,
-					},
-				],
-			};
-		} );
-	},
-
-	/**
-	 * Returns a normalized statsEmailsClick array, ready for use in stats-module
-	 *
-	 * @param   {Object} data   Stats data
-	 * @param   {Object} query  Stats query
-	 * @param   {number} siteId  Site ID
-	 * @param   {Object} site    Site object
-	 * @returns {Array}       Normalized stats data
-	 */
-	statsEmailsClick( data, query = {}, siteId, site ) {
-		if ( ! data || ! query.period || ! query.date ) {
-			return [];
-		}
-
-		const emailsData = get( data, [ 'posts' ], [] );
-
-		return emailsData.map( ( { id, href, date, title, type, clicks } ) => {
-			const detailPage = site ? `/stats/email/clicks/day/${ id }/${ site.slug }` : null;
 			return {
 				id,
 				href,
 				date,
 				label: title,
 				type,
 				value: clicks || '0',
+				opens: opens || '0',
+				clicks: clicks || '0',
```

Comment:
is `value` still required when we have `opens` and `clicks`?

Reply:
Yes, `value` is the one that's actually being used by the component. The other one is used for the additional column. When switching the columns' positions, I decided to keep both values here, just in case we ever need to change them around again.

Reply:
Nice 👍